### PR TITLE
[ViP-DeepLab] Adds support for SemKITTI-DVPS

### PR DESCRIPTION
**Pull request by joe-siyuan-qiao**
This PR adds support for SemKITTI-DVPS and adds more details to the markdown.

### CODE REVIEW

File: data/build_dvps_data.py [hunk 1]
```diff
@@ -90,6 +91,10 @@
 flags.DEFINE_string('output_dir', None,
                     'Path to save converted TFRecord of TensorFlow examples.')
 
+flags.DEFINE_integer('panoptic_divisor', default=0,
```
Comment (aquariusjay) [inline data/build_dvps_data.py:94]: default to the real value you use?

File: data/dataset.py [hunk 2]
```diff
@@ -89,6 +89,7 @@
 _KITTI_STEP = 'kitti_step'
 _MOTCHALLENGE_STEP = 'motchallenge_step'
 _CITYSCAPES_DVPS = 'cityscapes_dvps'
+_SEMKITTI_DVPS = 'semkitti_dvps'
```
Comment (aquariusjay) [inline data/dataset.py:92]: Add semantickitti paper reference in the docstring above.

File: g3doc/projects/vip_deeplab.md [hunk 3]
```diff
@@ -38,4 +57,8 @@ results, please use the following BibTeX entry.
 ```
 
 ### References
-Add some related works if any
+1. Dahun Kim, Sanghyun Woo, Joon-Young Lee, and In So Kweon. "Video panoptic
```
Comment (aquariusjay) [inline g3doc/projects/vip_deeplab.md:60]: Also add references to cityscapes, kitti, and semantic kitti.

**Review by aquariusjay — COMMENTED** [review]
Looks good on my side. Only a few nits. PTAL. Thanks, Siyuan!

**Comment by joe-siyuan-qiao** [comment]
@aquariusjay Thanks for the suggestions! The code has been revised accordingly.

File: g3doc/projects/vip_deeplab.md [hunk 4]
```diff
@@ -1,12 +1,31 @@
...
+
+## Prerequisite
+
+1. Make sure the software is properly [installed](../setup/installation.md).
+
+2. Make sure the target dataset is correctly prepared (e.g.,
+[SemKITTI-DVPS](https://github.com/joe-siyuan-qiao/ViP-DeepLab/tree/master/semkitti-dvps), [Cityscapes-DVPS](https://github.com/joe-siyuan-qiao/ViP-DeepLab/tree/master/cityscapes-dvps)).
+
+3. Download the pretrained Panoptic-DeepLab
+[checkpoints](./panoptic_deeplab.md), and update the
```
Comment (ClaireXie) [inline g3doc/projects/vip_deeplab.md:27]: Can you be more specific about which checkpoints? Cityscape or COCO? Maybe link the header session in https://github.com/google-research/deeplab2/blob/main/g3doc/projects/panoptic_deeplab.md instead.

File: g3doc/projects/vip_deeplab.md [hunk 5]
```diff
@@ -38,4 +57,20 @@ results, please use the following BibTeX entry.
 ```
 
 ### References
-Add some related works if any
```
Comment (ClaireXie) [inline g3doc/projects/vip_deeplab.md:41]: Also is there any plan to show some results(numbers and ckpts) as Huiyu did for max-deeplab?

**Review by ClaireXie — APPROVED** [review]
Looks great!

File: g3doc/projects/vip_deeplab.md (hunk 5 above)
Comment (joe-siyuan-qiao) [reply g3doc/projects/vip_deeplab.md:41]: I have very limited computation resources right now. I'll try to train some models for them.

**Comment by joe-siyuan-qiao** [comment]
@ClaireXie Thanks for the comments. I have revised the codes accordingly.

**Review by ClaireXie — APPROVED** [review]

**Review by YknZhu — APPROVED** [review]

**Pull request merged**